tutorials/animation/2d_skeletons.rst

```diff
@@ -3,238 +3,260 @@
 2D skeletons
 ============
 
-.. warning::
+What is cutout animation?
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    There are known issues with 2D skeletons on mobile and web platforms with the GLES2 renderer. We
-    recommend using the GLES3 renderer if your project relies on Skeleton2D for now.
+Traditionally, `cutout animation <https://en.wikipedia.org/wiki/Cutout_animation>`__
+is a type of `stop motion animation <https://en.wikipedia.org/wiki/Stop_motion>`__
+in which pieces of paper (or other thin material) are cut into special shapes
+and arranged in two-dimensional representations of characters and objects.
+Characters' bodies are usually made out of several pieces. The pieces are
+arranged and photographed once for each frame of the film. The animator moves
+and rotates the parts in small increments between each shot to create the
+illusion of movement when the images are played back quickly in sequence.
 
-Introduction
-------------
+Simulations of cutout animation can now be created using software as seen in
+`South Park <https://en.wikipedia.org/wiki/South_Park>`__ and `Jake and the Never
+Land Pirates <https://en.wikipedia.org/wiki/Jake_and_the_Never_Land_Pirates>`__.
 
-When working with 3D, skeletal deforms are common for characters and creatures
-and most 3D modelling applications support it. For 2D, as this function is not
-used as often, it's difficult to find mainstream software aimed for this.
+In video games, this technique has also become popular. Examples of
+this are `Paper Mario <https://en.wikipedia.org/wiki/Super_Paper_Mario>`__ or
+`Rayman Origins <https://en.wikipedia.org/wiki/Rayman_Origins>`__ .
 
-One option is to create animations in third-party software such as Spine or
-Dragonbones. From Godot 3.1 onwards, though, this functionality is supported
-built-in.
+Cutout animation in Godot
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Why would you want to do skeletal animations directly in Godot? The answer is
-that there are many advantages to it:
+Godot provides tools for working with cutout rigs, and is ideal for the workflow:
 
-* Better integration with the engine, so less hassle importing and editing from
-  an external tool.
-* Ability to control particle systems, shaders, sounds, call scripts, colors,
-  transparency, etc. in animations.
-* The built-in skeletal system in Godot is very efficient and designed for
-  performance.
+-  **The animation system is fully integrated with the engine**: This
+   means animations can control much more than just motion of objects. Textures,
+   sprite sizes, pivots, opacity, color modulation, and more, can all be animated
```

Comment:
You mention pivots, but they seem to be missing from this documentation - [they were covered in the original documentation](https://docs.godotengine.org/en/stable/tutorials/animation/cutout_animation.html).

Reply:
Oh right pivots are not necessary any more i don't think so, i'll remove it

Reply:
okay i did it